src/net.cpp

```diff
@@ -2356,6 +2359,13 @@ CNode::CNode(SOCKET hSocketIn, const CAddress& addrIn, const std::string& addrNa
     else
         LogPrint("net", "Added connection peer=%d\n", id);
 
+    for (size_t i = 0; i < allNetMessageTypesSize; i++) {
+        std::pair<uint64_t, uint64_t>& messageTypeSend = mapSendBytesPerMsgType[allNetMessageTypes[i]];
+        messageTypeSend.first = messageTypeSend.second = 0;
+        std::pair<uint64_t, uint64_t>& messageTypeRecv = mapRecvBytesPerMsgType[allNetMessageTypes[i]];
+        messageTypeRecv.first = messageTypeRecv.second = 0;
```

Comment:
Tiny thing here, but how about something like the following?
```suggestion
        mapSendBytesPerMsgType.insert(allNetMessageTypes[i], {0, 0});
        mapRecvBytesPerMsgType.insert(allNetMessageTypes[i], {0, 0});
```

Reply:
Way cleaner, I like that.